src/main/resources/application.yaml

```diff
@@ -1321,6 +1349,7 @@ case_document_am.url: ${CASE_DOCUMENT_AM_URL:http://localhost:4455}
 feature.save_correspondence: ${SAVE_CORRESPONDENCE:true}
 feature.covid19: ${COVID_19_NOTIFICATION_FEATURE:false}
 feature.secure-doc-store.enabled: ${SECURE_DOC_STORE_FEATURE:false}
+feature.postHearings.enabled: ${POST_HEARINGS_FEATURE:false}
```

Comment:
does this need to be post hearings b flag? As it is an LTA ticket

Reply:
reworked post hearing feature flags, not there are both post hearing and post hearing b flags